Error importing survey from Collect

![Screenshot_2021-02-10_14-50-40](https://user-images.githubusercontent.com/1219739/107518664-8d5c0900-6baf-11eb-882e-578c3af3131b.png)

This is the stack trace of the error: 
```
[2021-02-10T14:48:43.378] [ERROR] arena - Job NodeDefsImportJob - TypeError: client.tx is not a function
    at Module.insertNodeDef (/home/ste/dev/projects/openforis/arena/dist/jobThread.js:23556:73)
    at NodeDefsImportJob.addSpecifyTextAttribute (/home/ste/dev/projects/openforis/arena/dist/jobThread.js:22552:147)
    at NodeDefsImportJob.insertNodeDef (/home/ste/dev/projects/openforis/arena/dist/jobThread.js:22302:22)
    at async NodeDefsImportJob.insertNodeDefChildren (/home/ste/dev/projects/openforis/arena/dist/jobThread.js:22342:36)
    at async NodeDefsImportJob.insertNodeDef (/home/ste/dev/projects/openforis/arena/dist/jobThread.js:22284:29)
    at async NodeDefsImportJob.execute (/home/ste/dev/projects/openforis/arena/dist/jobThread.js:22216:5)
    at async NodeDefsImportJob._executeInTransaction (/home/ste/dev/projects/openforis/arena/dist/jobThread.js:15717:11)

```

The error is due to a change of the insertNodeDef function signature.

--- a/server/modules/collectImport/service/collectImport/metaImportJobs/nodeDefsImportJob.js
+++ b/server/modules/collectImport/service/collectImport/metaImportJobs/nodeDefsImportJob.js
@@ -159,10 +159,7 @@ export default class NodeDefsImportJob extends Job {
 
     Object.assign(
       nodeDefsUpdated,
-      await NodeDefManager.insertNodeDef(
-        { user: this.user, surveyId, cycle: Survey.cycleOneKey, nodeDef: nodeDefParam, system: true },
-        this.tx
-      )
+      await NodeDefManager.insertNodeDef({ user: this.user, surveyId, nodeDef: nodeDefParam, system: true }, this.tx)
     )
 
     let nodeDef = nodeDefsUpdated[nodeDefUuid]
@@ -483,11 +480,12 @@ export default class NodeDefsImportJob extends Job {
       }
       const qualifierNodeDefParam = _createNodeDef(parentNodeDef, NodeDef.nodeDefType.text, props, propsAdvanced)
       const qualifierNodeDefAndOthersUpdated = await NodeDefManager.insertNodeDef(
-        this.user,
-        this.surveyId,
-        Survey.cycleOneKey,
-        qualifierNodeDefParam,
-        true,
+        {
+          user: this.user,
+          surveyId: this.surveyId,
+          nodeDef: qualifierNodeDefParam,
+          system: true,
+        },
         this.tx
       )
       const qualifierNodeDefUuid = NodeDef.getUuid(qualifierNodeDefParam)

--- a/server/modules/nodeDef/manager/nodeDefManager.js
+++ b/server/modules/nodeDef/manager/nodeDefManager.js
@@ -1,6 +1,7 @@
 import * as R from 'ramda'
 import { db } from '@server/db/db'
 
+import * as Survey from '@core/survey/survey'
 import * as NodeDef from '@core/survey/nodeDef'
 import * as ObjectUtils from '@core/objectUtils'
 
@@ -24,7 +25,7 @@ export {
 // ======= CREATE
 
 export const insertNodeDef = async (
-  { user, surveyId, cycle, nodeDef: nodeDefParam, system = false, addLogs = true },
+  { user, surveyId, cycle = Survey.cycleOneKey, nodeDef: nodeDefParam, system = false, addLogs = true },
   client = db
 ) =>
   client.tx(async (t) => {

--- a/server/modules/survey/manager/surveyManager.js
+++ b/server/modules/survey/manager/surveyManager.js
@@ -84,10 +84,7 @@ export const insertSurvey = async (params, client = db) => {
           ),
         }
       )
-      await NodeDefManager.insertNodeDef(
-        { user, surveyId, cycle: Survey.cycleOneKey, nodeDef: rootEntityDef, system: true },
-        t
-      )
+      await NodeDefManager.insertNodeDef({ user, surveyId, nodeDef: rootEntityDef, system: true }, t)
     }
 
     // Update user prefs

